homeassistant/components/unifi/manifest.json

```diff
@@ -13,14 +13,16 @@
   "ssdp": [
     {
       "manufacturer": "Ubiquiti Networks",
-      "deviceType": "urn:schemas-upnp-org:device:InternetGatewayDevice:1",
       "modelDescription": "UniFi Dream Machine"
     },
     {
       "manufacturer": "Ubiquiti Networks",
-      "deviceType": "urn:schemas-upnp-org:device:InternetGatewayDevice:1",
       "modelDescription": "UniFi Dream Machine Pro"
-    }
+    },
+    {
+      "manufacturer": "Ubiquiti Networks",
+      "modelDescription": "UniFi Dream Machine SE"
```

Comment:
@bdraco it seems that the (new / EA) Dream Router is also missing. I own the Dream Router, can I help in any way to get the signature? Not sure how to gather the SSDP details...

Reply:
I think you can enable debug logging for ssd in ha to see everything

Reply:
@iMicknl 

`python3 -m netdisco dump`

Then follow the url to the .xml file.

Reply:
Weird, my Dream Router (192.168.1.1) doesn't show up, while all my other UPNPEntries work (Sonos mainly).